src/iotjs_binding.cpp

```diff
@@ -116,13 +119,17 @@ JObject::~JObject() {
 
 
 JObject& JObject::Null() {
-  static JObject null(JVal::Null(), false);
-  return null;
+  if (null == NULL) {
+    null = new JObject(JVal::Null(), false);
```

Comment:
@akosthekiss  I don't know much about NuttX, but hopefully I found [this commit](https://github.com/bherrera/NuttX/commit/15c1ace16cc4f7045ea28623936db23929631fea), after rebasing with his branch, i made it to build nuttx. I will write guild for this later on.